### ISSUE TITLE
(b/74108452): change the loss registration functions names to refer 

### ISSUE THREAD

**Issue by piascikj** [imdone/tensorflow#789]
to "loss functions" instead of distributions.  Following naming convention
of the loss function classes themselves.

----
*opened via [imdone.io](https://imdone.io) from a code comment on [4ac1fee7f13586ce6633a45bbe88592f605583e0](https://github.com/imdone/tensorflow/commit/4ac1fee7f13586ce6633a45bbe88592f605583e0) by A. Unique TensorFlower gardener@tensorflow.org*

----

https://github.com/imdone/tensorflow/blob/8cb6e535f6ea14b380aa86d425169adad682cb8c/tensorflow/contrib/kfac/python/ops/layer_collection.py#L1080-L1087
